### DISCUSSION TITLE
Please add support for Windows PE

### DISCUSSION THREAD

**Discussion by Lilianne-Blaze** [rustdesk/rustdesk#10244] [General]
Please add support for Windows PE, currently it's working but only partially.
When RustDesk is started in WinPE, you can connect and control desktop as usual, but you can't enter file transfer screen, it results in error "No active console user logged on, please connect and logon first."

Why I believe it's important:
RustDesk can be burned in a recovery Windows PE media, which a technician can ask a customer to boot from, and it just works, connects to internet and provides remote access. Right now it works, but without file transfer, which is a major inconvenience.

I forked and managed to trial-and-error a version that works normally, here's the important part:
https://github.com/Lilianne-Blaze/rustdesk/commit/863169c1215cbad18a4d081de2444d58195b0989

You can check it at
https://github.com/Lilianne-Blaze/rustdesk/tree/windows-pe-support
https://github.com/Lilianne-Blaze/rustdesk/releases/tag/1.3.5-1

I have close-to-zero knowledge of Rust, and I have no idea if it could cause side effects in environments other than Windows PE, so I assume it needs to be rewritten with some checks.
If needed you can detect typical WinPEs by checking for vars "USERNAME=SYSTEM" "USERPROFILE=X:\Users\Default", note the "X:" drive. It doesn't have to be "X:", but almost always is.

**Comment by SiLenCe-WB** [comment]
Any news on this? I have my own rustdesk server and I would prefer to use it over AnyDesk on Hiren's Boot. It would be nice if file transfer worked.

Also, the popup window to accept connections is not working either. If you don't input the password, you can't connect. I supposed this issue is also related to the file transfer window bug.
